docs/src/create-apps/hooks/hooks-comparison.md

```diff
@@ -51,6 +51,13 @@ If a `build` hook fails for any reason, the build is aborted and the deploy does
 Note that this only works for `build` hooks.
 If other hooks fail, the deploy still happens.
 
+### Timeout
+
+Build hooks automatically time out if they run for 1 hour.
+So if you accidentally add an unbroken loop, it gets cut off and you can continue with other activities.
+
+{{% legacy-regions featureIntro="Build hooks timing out" featureShort="this feature" level=4 %}}
```

Comment:
:nail_care:

To be aligned with the "deploy hook legacy note", this item should probably be one level higher.
That would also add it to the nav menu.
https://pr-2383-ax73k6a-652soceglkw4u.eu-3.platformsh.site/create-apps/hooks/hooks-comparison.html#supported-regions-1 

```suggestion
{{% legacy-regions featureIntro="Build hooks timing out" featureShort="this feature" level=3 %}}
```

Reply:
I don't think I agree. The current structure is:
* Build
  * Build timeout
    * Supported regions
 * Deploy 
   * Supported regions

There's another layer of structure in the build one. If it were level 3, it would apply to all of the build hook, but it's only about the timeout, so that's why it's structured under that.

I could see the idea of adding more structure to the deploy section, but I wouldn't remove a level under the build.